### PR TITLE
v8.2 release

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-44843
Staging - [Compat](https://deploy-preview-38--docs-entity-framework.netlify.app/#compatibility)
[What's New](https://deploy-preview-38--docs-entity-framework.netlify.app/whats-new/)
[Upgrade](https://deploy-preview-38--docs-entity-framework.netlify.app/upgrade/)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/upgrade.txt [hunk 1]
```diff
@@ -48,6 +48,20 @@ changes between the current and upgrade versions. For example, if you
 are upgrading the provider from v8.0 to v8.5, address all breaking changes from
 the version after v8.0, including any listed under v8.5.
 
+.. _entity-framework-breaking-changes-v8.2:
+
+Version 8.2.0 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Version 8.2.0 of the {+provider-short+} does not directly introduce any breaking
+changes. However, the underlying {+csharp-driver-short+} introduces many
```
Comment (mongoKart) [inline source/upgrade.txt:57]: Q; does 8.2 require C# v3.0? if so, might want to be explicit about that

**Review by mongoKart — APPROVED** [review]
LGTM w/ 1 suggestion

File: source/upgrade.txt (hunk 1 above)
Comment (jordan-smith721) [reply source/upgrade.txt:57]: I think it comes bundled with it. @damieng can you confirm?

Comment (damieng) [reply source/upgrade.txt:57]: It's a direct dependency, so not bundled as such but as soon as your add the MongoDB EF Core Provider NuGet is going to go get the MongoDB C# Driver 3.0 as well.

**Review by damieng — APPROVED** [review]

**Pull request merged**